app/helpers/check_answers_helper.rb

```diff
@@ -1,161 +1,218 @@
 module CheckAnswersHelper
-  EXCLUDED_QUESTIONS = %w[
-    medical_equipment
-    medical_equipment_type
-  ].freeze
-
-  SKIPPABLE_QUESTIONS = %w[
-    rooms_number
-    accommodation_cost
-    transport_type
-    transport_cost
-    offer_space_type
-    space_cost
-    offer_care_qualifications
-    care_cost
-    offer_staff_type
-    offer_staff_charge
-    construction_services
-    construction_services_other
-    construction_cost
-    it_services
-    it_services_other
-    it_cost
-  ].freeze
-
-  def items
-    items = questions.map do |question|
-      # We have answers as strings and hashes. The hashes need a little more
-      # work to make them readable.
+  def sections
+    %w[accommodation transport space staff care services other_support location business_details contact_details]
+  end
 
-      next if skip_question?(question)
+  def match_number_to_list_item(numbers, list_item)
+    value = numbers.with_indifferent_access["#{list_item.downcase.gsub(' ', '_')}_number"]
+    value.present? ? number_with_delimiter(value, delimiter: ",") : "0"
+  end
 
-      next transport_type if question.eql?("transport_type")
-      next offer_care_qualifications if question.eql?("offer_care_qualifications")
-      next offer_staff_type if question.eql?("offer_staff_type")
-      next link_to_parent_page(question, "offer_staff_type") if question.eql?("offer_staff_charge")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_services_other")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_cost")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_services_other")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_cost")
-      next link_to_parent_page(question, "transport_type") if question.eql?("transport_cost")
-      next link_to_parent_page(question, "offer_care_qualifications") if question.eql?("care_cost")
-      next link_to_parent_page(question, "rooms_number") if question.eql?("accommodation_cost")
-      next link_to_parent_page(question, "offer_space_type") if question.eql?("space_cost")
+  def staff_numbers
+    {
+      cleaners_number: session[:offer_staff_number][:cleaners_number],
+      developers_number: session[:offer_staff_number][:developers_number],
+      medical_staff_number: session[:offer_staff_number][:medical_staff_number],
+      office_staff_number: session[:offer_staff_number][:office_staff_number],
+      security_staff_number: session[:offer_staff_number][:security_staff_number],
+      trainers_or_coaches_number: session[:offer_staff_number][:trainers_number],
+      translators_number: session[:offer_staff_number][:translators_number],
+      other_staff_number: session[:offer_staff_number][:other_staff_number],
+    }.with_indifferent_access
+  end
```

Comment:
I think this has the same effect, and makes it a little clear what difference is between the two hashes:
```ruby
def staff_numbers
  staff_numbers = session[:offer_staff_number]
  staff_numbers[:trainers_or_coaches_number] = staff_numbers.delete(:trainers_number)
  staff_numbers.with_indifferent_access
end
```

Reply:
Thanks Rob, we went with this suggestion 👍